lib/smart_answer/calculators/pay_leave_for_parents_calculator.rb

```diff
@@ -11,6 +11,7 @@ class PayLeaveForParentsCalculator
     attr_accessor :mother_earned_more_than_lower_earnings_limit
     attr_accessor :mother_worked_at_least_26_weeks
     attr_accessor :mother_earned_at_least_390
+    attr_accessor :salary_1_66_weeks
```

Comment:
I've changed the commit titled "Store salary_1_66_weeks in attribute on calculator" (32e81e7ee639eb535326ae7a6c5e3af5e2377bdd) so that it now just removes the `save_input_as` and no longer adds the redundant `on_response` or `salary_1_66_weeks` calculator attribute in a new version of the commit titled "Remove redundant save_input_as for salary_1_66_weeks" (a6245c77b92289b94fb9b03d0b2ef7777379399e).